packages/tictactoe/package.json

```diff
@@ -8,6 +8,20 @@
   "resolutions": {
     "firebase/**/grpc": "^1.16.1"
   },
+  "scripts": {
+    "start": "node scripts/start.js",
+    "build": "node --max-old-space-size=4076 scripts/build.js",
+    "test": "run-s prettier:check test:contracts 'test:app --all'",
+    "test:ci": "DEV_GANACHE_PORT=8504 run-s prettier:check build 'test:contracts --all --ci' 'test:app --all --ci --runInBand'",
```

Comment:
@andrewgordstewart Should `DEV_GANACHE_PORT` setting get defined in one of the config files?

Reply:
Maybe, but it's outside of the scope of this pr. (Only prettier:check was added)